webapp/packages/core-blocks/src/Split/Pane.tsx

```diff
@@ -11,9 +11,12 @@ import { observer } from 'mobx-react-lite';
 import { s } from '../s';
 import { useS } from '../useS';
 import style from './Pane.m.css';
+import { useSplit } from './useSplit';
 
 export const Pane = observer<PaneProps>(function Pane({ className, ...rest }) {
   const styles = useS(style);
+  const split = useSplit();
+  const hiddenContent = rest.main && split.state.mode === 'minimize';
 
```

Comment:
do we want to also hide content of second pane when mode is 'maximize'

Reply:
I dont think so. We should see that something left here we can work with. Otherwise it is easy to get lost in the app